{
  "modified": "2024-10-16T14:37:12Z",
  "published": "2024-10-16T14:37:12Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2024-4838"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "bsodinator"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/bsodinator/MAL-2024-4838.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "107f3b559bd418e1bb1312879b4012d3d2ee0d6ffea679d14097f0d8e6eb0dbe",
        "import_time": "2024-10-24T00:58:51.358289706Z",
        "id": "RLUA-2024-07900",
        "modified_time": "2024-10-16T14:37:12Z"
      }
    ]
  }
}
